locales/de-global.json5

```
@@ -119,11 +132,114 @@
   today: 'Heute',
   more: 'Mehr',
   less: 'Weniger',
-  search: 'Suche',
+  search: {
+    articles: {
+      placeholder: 'Artikel suchen...',
+      ariaLabel: 'Suchen Sie nach Artikeln und Dokumentationen',
+    },
+  },
```

this is unused as far as I can see 🤔
So we can either delete it completely of revert it to it's original value

```suggestion
  search: 'Suche',
```

---

You're right. It is no longer being used. 
IT is because before making the final feature, I built a prototype feature to play around. Unfortunately, this prototype left a lot of garbage in the code. I apologize for missing this step of cleaning